app/logic/Calendar/SQL/createDatabase.ts

```diff
@@ -132,11 +132,13 @@ export const calendarDatabaseSchema = sql`
     "eventID" INTEGER not null,
     "emailAddress" TEXT not null,
     "name" TEXT,
-    -- 1 = accepted
-    -- 2 = rejected
-    -- 3 = tentative accepted
-    -- null = not responded
-    "confirmed" INTEGER default null,
+    -- 0 = unknown
+    -- 1 = organizer
+    -- 2 = tentative
+    -- 3 = accepted
+    -- 4 = rejected
+    -- 5 = not responded
+    "response" INTEGER default 0,
```

Comment:
We don't have a database migration for this, so this would trash all existing profiles and throw errors for all testers and users. Even those who have shortly tried Mustang in the past and will try it again. So, please avoid database changes. If you do, please start a migration.

Reply:
I'm changing this column name back.
I understand that the values changed and will now be wrong for existing databases, but nobody other than you tested this feature, because it didn't exist.
In contrast, a DB schema change will break the entire app for 100% everybody.

Reply:
When did you start requiring database migrations? Some warning would have been nice.

Reply:
Since always. I already told you when you added the current fields (`respondToOrganizer`). That caused tons of drama for our testers.

Reply:
My plan is to remove those fields for emails, and add a generic JSON column to all tables. This is particularly important for email, where we have lots of other structured data potentially in the emails that we want to save, and we cannot make columns for each.

The thing stopping me is exactly that migration necessary.

Reply:
I'll have to take your word for it, as I can't find any record of it.